scalability_and_performance/ztp-zero-touch-provisioning.adoc

```diff
@@ -1,21 +1,20 @@
 // CNF-1500 assembly
 [id="ztp-zero-touch-provisioning"]
-= Zero touch provisioning
+= Deploying distributed units at scale in a disconnected environment
 include::modules/common-attributes.adoc[]
 :context: zero-touch-provisioning
```

Comment:
Either way, the context and/or the anchor ID need [to change](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata).

> Context used for identifying headers in modules that is the same as the anchor id.